content/blog/2021/05/2021-05-15-configure-plugins-with-jcasc.adoc

```diff
@@ -0,0 +1,115 @@
+
+'''
+
+.Configuring your first plugin using JCasC (Video Demo)
+video::YeWhqLPjvMs[youtube, width=852, height=480]
+
+'''
+
+== Overview
+
+So, these are the steps we will be following to achieve our aim:
+
+. Configure the plugin on the UI
+. Download the configuration
+. Update JCasC file locally
+. Load the jenkins.yaml file on the Jenkins server
+. Verify the changes on the UI
+
+'''
+
+== 1. Configure the plugin on the UI
+
+- Make sure you have the `Configuration as Code` plugin installed, and for the demo let’s also install the `View Job Filters` plugin.
+- Let’s create a view by clicking on the `New View` option on the left side of the Dashboard.
+- Give it a name (say, “testView”) and set its type to `List View`, and click on the `OK` button.
+
+.Creating the View
+image::/images/post-images/2021-05-15-configure-plugins-with-jcasc/naming-the-view.png[Naming the View, width=700px]
+
+- Now click on `Add Job Filter` to add any kind of filter, so let’s select `Build Duration Filter` and fill the field with any value (say, "60" minutes),
+
+.Adding filter to the view
+image::/images/post-images/2021-05-15-configure-plugins-with-jcasc/add-filters-to-view.png[Add Filters, width=700px]
+
+- Click on `Apply` > `Save`.
+- To view the full configuration, check out your main "jenkins.yaml" configuration file, by clicking on `Manage Jenkins` > `Configuration as Code` > `View Configuration`
+- Go to the `views` section in this YAML file to see details related to the view,
+
+.Here, details regarding the view (which we just created) is visible 
+image::/images/post-images/2021-05-15-configure-plugins-with-jcasc/yaml-file-on-jenkins-ui.png[YAML file on Jenkins UI, width=750px]
+
+'''
+
+== 2. Download the Configuration
+
+- Now that you have successfully configured your plugin by UI, let’s download the configuration by going to `Manage Jenkins` on the Dashboard, then click on `Configuration as Code` under "System Configuration".
+- Now click on the `Download Configuration` button to save the configuration file locally.
+
+NOTE: The field `Path or URL` can be left blank for now, it will be used in upcoming steps.
+
+.Downloading the Configuration
+image::/images/post-images/2021-05-15-configure-plugins-with-jcasc/download-config-button.png[Download Configuration, height=400px]
+
+'''
+
+== 3. Update JCasC file locally
+
+- Add some changes in your downloaded copy of the "jenkins.yaml" file, to see those changes being automatically reflected on the UI.
+- For demo purposes, let’s change the `name` to “YoutubeDemoView” and set the `buildDurationMinutes` as "55".
+
+.Changing the View details locally
+image::/images/post-images/2021-05-15-configure-plugins-with-jcasc/yaml-file-on-local-text-editor.png[YAML file on Text Editor, width=750px]
+
+- Save the file.
+
+'''
+
+== 4. Load the jenkins.yaml file on the Jenkins server
+
+- Now to update the local changes done in the "jenkins.yaml" file, go to `Manage Jenkins` > `Configuration as Code`,
```

Comment:
Reload the file instead, replace is not a commonly used functionality

Reply:
So should I completely remove this "replace" file instruction and add "reload" the file instruction,
Or
Add both of these options and mark the "reload" option to be the recommended one?